EngineLayer/FdrAnalysis/PEPValueAnalysisGeneric.cs

```diff
@@ -240,17 +280,18 @@ private static float GetSSRCalcHydrophobicityZScore(PeptideSpectralMatch psm, Pe
 
             if (d.ContainsKey(psm.FullFilePath))
             {
-                int time = (int)Math.Round(psm.ScanRetentionTime, 0);
+                int time = (int)(2 * Math.Round(psm.ScanRetentionTime / 2d, 0));
                 if (d[psm.FullFilePath].Keys.Contains(time))
                 {
                     double predictedHydrophobicity = calc.ScoreSequence(Peptide);
                     hydrophobicityZscore = Math.Abs(d[psm.FullFilePath][time].Item1 - predictedHydrophobicity) / d[psm.FullFilePath][time].Item2;
                 }
             }
 
-            if (double.IsNaN(hydrophobicityZscore) || double.IsInfinity(hydrophobicityZscore))
+            double maxHydrophobicityZscore = 10;
```

Comment:
Can you comment on why this is 10? Instead of, say, 100? If it's arbitrary, note it.

Reply:
1 Z is one standard deviation. 10 standard deviations is lots.